.github/workflows/release.yml

```diff
@@ -4,6 +4,11 @@ on:
     push:
         branches:
             - main
+        paths:
+            - "packages/*.ts"
+            - "packages/*.tsx"
+            - "packages/*.scss"
+            - "yarn.lock"
```

Comment:
Jeg lurer på om jeg ville latt Lerna avgjøre om det kjøres release eller ikke (slik dte er i dag). Er fort gjort at man overser en endring som vanligvis ville ført til en release når man velger spesifikke paths, og `yarn install`/`lerna version` tar bare ett minutt å kjøre.

Det man _bør_ fikse om man lar dagens løsning være er at workflowen blir merket som feilet dersom det ikke er noen pakker som skal releases. Dette skyldes at `lerna version` avslutter med kode `1` om den ikke finner noe som skal versjoneres, men det bør være mulig å få GH Actions til ikke å regne dette som at flyten feiler.

Reply:
Gjorde et forsøk på exit-kodeworkaround nå. Noe relevant dokumentasjon:

- https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs-for-composite-actions
- https://docs.github.com/en/actions/learn-github-actions/expressions
- https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsif
- https://github.com/lerna/lerna/issues/2161#issuecomment-650873741